app/utils/sql/SqlInterpolation.scala

```diff
@@ -53,6 +53,9 @@ case class SqlToken(sql: String, values: List[SqlValue] = List()) {
   def debugInfo: String = {
     // The debugInfo should be pastable in an SQL client
     val parts = sql.split("\\?", -1)
+    if (parts.tail.length != values.length) {
+      println(s"[$sql] {${values}}")
```

Comment:
println should not stay in here. maybe return this instead?

Reply:
gone